includes/class-sensei-course.php

```diff
@@ -408,6 +421,14 @@ private function display_courses_navigation( WP_Screen $screen, array $tabs ) {
 					<?php
 				}
 				?>
+				<?php
+				if ( ! empty( $navigation_sidebar ) ) {
+					?>
+					<div class="sensei-custom-navigation__separator"></div>
+					<?php
+					echo $navigation_sidebar; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Content escaped in filter.
```

Comment:
Maybe a silly question, but...wouldn't it be better to use an action here instead of a filter?

We could try to detect if there are any actions using `has_action` (to add the separator), and we wouldn't need to echo raw HTML here. :thinking: 

Just an idea. I don't know if it is a good one, though. :)

Reply:
I thought about this option, but I ended up going with the filter, in case we have a case in the future to filter the filtered content, like removing the link. But thinking more, we could use the `remove_action` if we wanted. 🤔

I'll try to implement it very quickly now. If I'm not able to finish it today, I do it tomorrow.

Reply:
Yeah, I'm not sure about it either to be honest. :upside_down_face: 

Things would be clearer if instead of concatenating HTML we returned an e.g. array of links to be managed, like we did for tabs. Then I think filters would make more sense.

But...here, concatenating HTML just to echo it? I think actions would be a better deal. :thinking:

Reply:
Fernando, I was checking to implement it, but we add the hook anyway and we have the check inside the hook to see if we will add the extra content or not. So because we need that separator if we have something, I think it will be hard to implement it.

So I'm merging it, but if you have another idea, feel free to suggest that I can implement tomorrow.

> Things would be clearer if instead of concatenating HTML we returned an e.g. array of links to be managed, like we did for tabs. Then I think filters would make more sense.

About this, I think keeping it open is better for more flexibility if we want to introduce other things.